### PR TITLE
DATAES-207 fetch indices by alias

### PR DESCRIPTION
**Pull request by dandonovsc**

### CODE REVIEW

File: src/main/java/org/springframework/data/elasticsearch/core/ElasticsearchOperations.java [hunk 1]
```diff
@@ -558,4 +558,6 @@
 
 
 	<T> T query(SearchQuery query, ResultsExtractor<T> resultsExtractor);
+
+	Set<String> queryByAlias(String indexName);
```
Comment (martin-g) [inline src/main/java/org/springframework/data/elasticsearch/core/ElasticsearchOperations.java]: s/indexName/aliasName/

**Comment by dandonovsc** [comment]
Is there an issue to be corrected here on order to process forward?

**Comment by pivotal-issuemaster** [comment]
@dandonovsc Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-data-elasticsearch&pullRequestId=124) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by sothawo** [comment]
Superseded by #486

**Pull request closed without merging**